src/web/client/common/constants.ts

```diff
@@ -29,34 +29,65 @@ export const PROVIDER_ID = "microsoft";
 export const SCOPE_OPTION = "//.default";
 export const BAD_REQUEST = 'BAD_REQUEST';
 
+export enum HttpMethod {
+    PATCH = 'PATCH',
+    GET = 'GET'
+}
+
+export enum VscodeUrlPathEntityName {
```

Comment:
Please add comments to explain significance of these enums.

e.g.
1. This is the simplified/generic name of the entity passed in the URL by the clients
2. This is the entity name in the schema file. Schema file contains a lot of properties about how to fetch/save the entity and how to display it in the file explorer.
3. These are the OData names of the resources required for fetch and save calls.

Reply:
These names should be self explanatory. We should update these, if they are not clear. Will add the comments as well here.

Reply:
TODO: Put detailed comment on logic